donate/payments/tests/test_views.py

```diff
@@ -450,7 +449,9 @@ def test_subscription_data_submitted_to_braintree(self):
             'merchant_account_id': 'usd-ac',
             'payment_method_token': 'payment-method-1',
             'price': 50,
-            'first_billing_date': date(2019, 7, 26),
+            'options': {
```

Comment:
Based on this message from Braintree support:

"I’m Shaquille with Braintree Technical Support, happy to help.
I noted in the subscription examples provided the subscription was created with a specified start date in the future. This creates a subscription that remains Pending until the first billing date when the customer is charged and the subscription becomes Active. Do note, you can [create a subscription to start billing immediately](https://developer.paypal.com/braintree/docs/reference/request/subscription/create#set-start-date). This revision can be made to the plan that subscriptions are based on, or in the subscription.create() call itself."

It sounds like we are given two options when creating a new subscription: 
- Set the date and time for the subscription ourselves, which is what we were originally doing
- Or set the subscription to start immediately, which is what this PR is introducing. 

The reason for the change being that if a user submits a recurring donation past a certain time (around 7:30PM EST according to Jackie Lu) the user experiences a delay in recieving a donation receipt. 

Will Easton has also confirmed that when the user makes a donation at this time, the donation is set to "Pending" in braintree until the following day.

We believe this is due to differences in time zone between our application and Braintree. 

I believe that setting the donation to begin immediately will resolve this problem as it removes timezones from the equation.